src/init.cpp

```diff
@@ -1070,7 +1070,15 @@ bool AppInit2(Config &config, boost::thread_group &threadGroup, CScheduler &sche
                 delete pcoinsTip;
                 delete pcoinsdbview;
                 delete pcoinscatcher;
+                /**
+                 * pblocktree provices access to the block metadata database that will be used under the
+                 * current configuration settings for this node
+                 */
                 delete pblocktree;
+                /**
+                 * pblocktreeother provices access to some other block metadata database
```

Comment:
I have no clue what 'some other block metadata database' is, TBH :D
I am still confused by this.